client/src/components/biodata/BiodataList.tsx

```
@@ -0,0 +1,89 @@
+              <div className="w-full md:text-center p-4 bg-violet-900 rounded-md">
+                <p className="text-violet-100 bg-violet-950 font-bold text-xl rounded-lg border text-center p-2">
+                  Biodata - {user._id}
+                </p>
+                <div className=" p-2 text-lg md:text-xl">
+
+                <p className="text-white py-1 font-bold rounded-md">বয়স : {user.name}</p>
+                <p className="text-white py-1 font-bold rounded-md">উচ্চতা : {user.height}</p>
+                <p className="text-white py-1 font-bold rounded-md">পেশা : {user.work}</p>
+                </div>
+                <button className="bg-white py-2 px-4 rounded-full text-lg text-purple-800 hover:text-pink-900 hover:px-6 font-bold mt-2" onClick={() => showDetails(user._id)}>
+                  View Profile
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)